Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py

```diff
@@ -730,29 +730,53 @@ def get_a_list_of_unanalyzed_abuse_mailbox_campaigns_command(client, args):
     return command_results
 
 
-def generate_threat_incidents(threats, current_iso_format_time):
+def generate_threat_incidents(client, threats, current_iso_format_time):
     incidents = []
     for threat in threats:
-        incident = {"dbotMirrorId": str(threat["threatId"]), "name": "Threat", "occurred": current_iso_format_time,
-                    'details': "Threat"}
+        threat_details = None
+        try:
+            threat_details = client.get_details_of_a_threat_request(threat["threatId"])
```

Comment:
Do you not have all the details in the threat? Why do we need this request?

Reply:
No- the request to /threats only fetches the new threat IDs - we then have to fetch the details - same goes for cases and campaigns